### DISCUSSION TITLE
Personalized Labels/Colors on Dockge Home Page

### DISCUSSION THREAD

**Discussion by mahmoudhamadeh** [louislam/dockge#558] [Feature Request]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

UI Feature

### 🔖 Feature description

Add the ability to include personalized labels on the Dockge home page to help users identify which instance is currently open in a multi-instance Dockge environment. These labels can include details such as "server name," "hardware name," or any custom identifier that assists in distinguishing between different instances.

### ✔️ Solution

When using multiple Dockge instances, especially in setups involving a stack of Raspberry Pi's, it becomes challenging to quickly identify which instance is currently open. Currently, I have to rely on IP addresses in the address bar or remember the stack configuration for each Dockge instance, which can be confusing and error-prone. These proposed labels should be visible next to the Dockge logo or the Home icon, providing an immediate visual identifier for the current instance. Additionally, these custom labels should be referenced in the "Dockge Agents" list for consistency and ease of identification. A mock-up or annotated screenshot indicating where the label would be displayed included herewith.

![image](https://github.com/louislam/dockge/assets/26927138/e37aa8ce-cbe9-461a-bc4d-88a99d42d8a5)

### ❓ Alternatives

Provide an option to customize the color theme of the Dockge interface for each instance. This allows users to visually differentiate between instances based on color themes.

### 📝 Additional Context

That was it, thanks!

**Comment by ElectronicBattle** [comment]
Also see 
https://github.com/louislam/dockge/discussions/345
https://github.com/louislam/dockge/pull/414

This request is similar to those two but not the same.  Rapid ID of whether a container is "development" or "production", either handled locally or via a dockge agent from a different machine,  would be very handy - either by name or colour or tag or label etc.
